test_scripts/API/VehicleData/common.lua

```diff
@@ -75,7 +75,7 @@ m.vd = {
   eCallInfo = "VEHICLEDATA_ECALLINFO",
   airbagStatus = "VEHICLEDATA_AIRBAGSTATUS",
   emergencyEvent = "VEHICLEDATA_EMERGENCYEVENT",
-  -- clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS", -- disabled due to issue: https://github.com/smartdevicelink/sdl_core/issues/3460
+  clusterModeStatus = "VEHICLEDATA_CLUSTERMODESTATUS",
```

Comment:
It looks like we also need an update in https://github.com/smartdevicelink/sdl_atf_test_scripts/tree/develop/test_scripts/API/VehicleData/GenericNetworkSignalData/commonGenericNetSignalData.lua#L617

Reply:
This change would only applicable in case the name of parameter would be the same across all RPCs.
However we suppose Core should handle different names of the parameter correctly.